.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -11,67 +11,70 @@ assignees: ''
+1. **Identify Pull Requests**: Review any pull requests (PRs) that need to be backported to the release branch. These should be PRs that contain strictly bug fixes and nothing else.
+2. **Backport Pull Requests**: For PRs merged into the `main` branch that need to be applied to the release branch (e.g., `release/2.1.x`), comment on the merged PR with `/backport to release/2.1.x`. This comment will trigger a GitHub Action to create a new PR with the same changes for the `release/2.1.x` branch.
+3. **Merge PRs**: Once all necessary PRs are created, merge them into the `release/2.1.x` branch.
 
 ## Step 2: Code and Documentation Updates
 
-### Create Pull Request:
-
-  - [ ] From the release branch (e.g., `release/2.1`), create a new temporary branch for your release (e.g., `release-notes/2.1.1`).
-  - [ ] Update version references in the documentation. Refer to [this PR](https://github.com/OrchardCMS/OrchardCore/pull/17065/files) for an example. Version Updates Checklist:
-     - **Update `OrchardCore.Commons.props`**: Set `<VersionSuffix></VersionSuffix>` to the new version you're preparing for release.
-     - **Update Module Versions**: Modify `src/OrchardCore/OrchardCore.Abstractions/Modules/Manifest/ManifestConstants.cs` to reflect the new version.
-     - **Release Notes**: Finalize the release notes in the documentation, including:
-       - Highlights and goals of the release.
-       - Prerequisites for running the new version.
-       - Upgrade steps and any breaking changes.
-     - **Update Documentation Navigation**: Add the release notes page to `mkdocs.yml` navigation and remove it from `not_in_nav`.
-     - **Version Mentions**: Update all references to the new version throughout the documentation, including:
-       - [Status in the root README](https://docs.orchardcore.net/en/latest/#status)
-       - CLI templates and commands.
-       - Relevant guides, such as the [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/guides/decoupled-cms/) guide.
-  - [ ] Create a **Documentation PR** titled "Release with the new version number" (e.g., `Release 2.1.1`) from the documentation branch (e.g., `release-notes/2.1.1`) into the release branch (e.g., `release/2.1`)
-  - [ ] Merge the Documentation PR.
-  - [ ] In GitHub, manually run the `Preview - CI` workflow on your branch (NOT `main`). This will release a new preview version on CloudSmith for testing.
+- [ ] From the release branch (e.g., `release/2.1.x`), create a new temporary branch for the version (e.g., `release/2.1.1`).
+- [ ] Update version references in the documentation and code. Refer to [this PR](https://github.com/OrchardCMS/OrchardCore/pull/17065/files) for an example. The easiest is to start with a search & replace on the previous version. Version Updates Checklist:
+  - **Update `OrchardCore.Commons.props`**: Set `<VersionPrefix></VersionPrefix>` to the new version you're preparing for release.
```

Comment:
You don't actually need to set the `VersionSuffix`, ever (I don't really know why, though). It wasn't set for the last few releases and the version number correctly doesn't include it.

Reply:
You don't, but you should. If you follow the step, we should release a preview of this version to test it out, when you do this the preview CLI will create a preview on cloud smith with the correct version instead of creating a preview with the old version.

Reply:
That's not what I've seen. While `VersionSuffix` was kept as `preview` throughout, the preview package published to Cloudsmith (run for the version branch), used the version number `2.1.6-preview-18513`, see [here](https://github.com/OrchardCMS/OrchardCore/actions/runs/13425099869/job/37506426047). And when published to NuGet, the workflow generated `2.1.6` see [here](https://github.com/OrchardCMS/OrchardCore/actions/runs/13425524908/job/37507630862). So, the existence of `VersionSuffix` doesn't affect the non-preview release.

Note that I'm talking about Version**Suffix**. What you have to set, and what the guide now includes, is Version**Prefix**, what I've changed to `2.1.6`.

Reply:
okay.